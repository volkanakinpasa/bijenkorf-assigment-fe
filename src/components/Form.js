import '../styles/form.scss';

import SearchBox from './SearchBox';

function Form() {
  return (
    <section className="mx-auto form-section">
      <form className="w-full" role="search">
        <SearchBox
          onSearchInputChange={(e) => {
            console.log(e.target.value);
          }}
        />
      </form>
    </section>
  );
}

export default Form;
