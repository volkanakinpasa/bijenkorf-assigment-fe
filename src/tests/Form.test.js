import Form from '../components/Form';
import { render } from '@testing-library/react';

describe('Form tests', () => {
  it('creates a snapshot', () => {
    const { asFragment } = render(<Form />);
    expect(asFragment()).toMatchSnapshot();
  });
});
