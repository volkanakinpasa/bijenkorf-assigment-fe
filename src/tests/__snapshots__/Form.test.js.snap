// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form tests creates a snapshot 1`] = `
<DocumentFragment>
  <section
    class="mx-auto form-section"
  >
    <form
      class="w-full"
      role="search"
    >
      <div
        class="input-container relative"
      >
        <input
          autocomplete="off"
          class="search"
          data-testid="search"
          name="search"
          placeholder="Zooken"
          type="search"
        />
        <button
          aria-label="Zooken"
          type="submit"
        >
          <img
            src="search-icon.svg"
          />
        </button>
      </div>
    </form>
  </section>
</DocumentFragment>
`;
